aodndata/srs/srs_oc_gridded.py

```diff
@@ -14,32 +15,21 @@
 OC_GRIDDED_PREFIX_PATH = 'IMOS/SRS/OC/gridded'
 
 RJOHNSON_FILE_PATTERN = re.compile(r"""
-                                ^(?P<data_parameter_code>A|S)
+                                (?P<data_parameter_code>A|S)
```

Comment:
Not sure what the best practice is with the carets when using re.match... they seem redundant, but the pattern is separate from the match function, so I wonder if we should keep them in?